### PR TITLE
DBZ-1304 Fix release pipline

### PR DESCRIPTION
**Pull request by jpechane**
https://github.com/debezium/debezium/pull/899

### CODE REVIEW

File: postgres/10-alpine/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM postgres:10-alpine
 
 LABEL maintainer="Debezium Community"
-ARG USE_POSTGIS=true
+ARG USE_POSTGIS=false
```
Comment (gunnarmorling) [inline postgres/10-alpine/Dockerfile]: Why is this changing?

Comment (jpechane) [reply postgres/10-alpine/Dockerfile]: @gunnarmorling Sorry, I forgot to link the issue - https://issues.jboss.org/browse/DBZ-1307

Comment (gunnarmorling) [reply postgres/10-alpine/Dockerfile]: Ok, thanks. So are you planning to re-enable this in a follow-up as per "We need to make the package from source now to be able to build PostGIS"?

Comment (jpechane) [reply postgres/10-alpine/Dockerfile]: @gunnarmorling Yes, just it has lower priority now. I am no Alpine expert so we can expect some lerning curve.

Comment (gunnarmorling) [reply postgres/10-alpine/Dockerfile]: +1 Can you only file a JIRA issue so it's tracked. Thanks!

Comment (jpechane) [reply postgres/10-alpine/Dockerfile]: @gunnarmorling I am now a little bit confused - Jira issue for what? I've already logged one

Comment (gunnarmorling) [reply postgres/10-alpine/Dockerfile]: Ah, so I was confused, too :) I thought 1307 was to document the disablement of PostGIS under Alpine, whereas it's meant for re-enabling this. So this warrants a mention in the release notes then.

**Comment by renatomefi** [comment]
Sorry for missing this out, somehow it was a blindspot for me!

**Pull request merged**